examples/UbidotsGetValue/UbidotsGetValue.ino

```diff
@@ -1,24 +1,24 @@
 #include <UbidotsFONA.h>
 #include <SoftwareSerial.h>
 
-
```

Comment:
Add in all the examples how to activate debug messages

Reply:
The debug method was included.